tests/test_selfattention.py

```diff
@@ -24,25 +24,29 @@
 from monai.utils import optional_import
 
 einops, has_einops = optional_import("einops")
+xops, has_xformers = optional_import("xformers.ops")
 
 TEST_CASE_SABLOCK = []
 for dropout_rate in np.linspace(0, 1, 4):
```

Comment:
I'd like to suggest using itertools.product here rather than the nested for loops